docs/custom-pools.md

```diff
@@ -66,6 +66,10 @@ worker   rendered-worker-6db67f47c0b205c26561b1c5ab74d79b   True      False
 
 The example above makes an `infra` pool that contains all of the MachineConfigs used by the `worker` pool.
 
+### Ordering the custom pool and resulting MachineConfigs
```

Comment:
We are changing this behaviour in 4.15, see https://issues.redhat.com/browse/MCO-641

Happy to help re-frame this.